### PR TITLE
bobhelper 0.1.0 (new cask)

### PR DESCRIPTION
**Pull request by mogeko**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/b/bob-helper.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  desc "Helper tool designed for Bob to solve the shortcut key issue"
+  homepage "https://bobtranslate.com/guide/advance/bobhelper.html"
+
+  depends_on macos: ">= :catalina"
+
+  app "BobHelper.app"
+
+  # No zap stanza required
+
+  caveats "Using BobHelper requires Bob version 1.11.0 or above."
```
Comment (bevanjkay) [inline Casks/b/bob-helper.rb:17]:
```suggestion
```

Caveats are reserved specifically for Homebrew-related installation notes. This note applies whether Homebrew is used to install the application or not.

File: Casks/b/bob-helper.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+cask "bob-helper" do
```
Comment (bevanjkay) [inline Casks/b/bob-helper.rb:1]:
```suggestion
cask "bobhelper" do
```

Please also update the filename to match.

File: Casks/b/bobhelper.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+cask "bobhelper" do
+  version :latest
+  sha256 :no_check
+
+  url "https://release.ripperhe.com/BobHelper/latest/BobHelper.dmg",
+      verified: "release.ripperhe.com/BobHelper/latest/"
```
Comment (khipp) [inline Casks/b/bobhelper.rb:6]:
```suggestion
  version "0.1.0"
  sha256 "b6e4e576a9e47cb956b4f3f475caa5619533ff6f5eaf237a8992894ec8ba5b5b"

  url "https://release.ripperhe.com/BobHelper/tag/v#{version}/BobHelper.zip",
      verified: "release.ripperhe.com/BobHelper/"
```

File: Casks/b/bobhelper.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  name "BobHelper"
+  desc "Helper tool designed for Bob to solve the shortcut key issue"
+  homepage "https://bobtranslate.com/guide/advance/bobhelper.html"
+
```
Comment (khipp) [inline Casks/b/bobhelper.rb:10]:
```suggestion

  livecheck do
    url "https://appcast.ripperhe.com/bobhelper/appcast.xml"
    strategy :sparkle, &:short_version
  end

```

File: Casks/b/bobhelper.rb [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  depends_on macos: ">= :catalina"
+
+  app "BobHelper.app"
+
```
Comment (khipp) [inline Casks/b/bobhelper.rb:14]:
```suggestion

  uninstall quit:       "com.hezongyidev.BobHelper",
            login_item: "BobHelper"

```

File: Casks/b/bobhelper.rb [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  # No zap stanza required
```
Comment (khipp) [inline Casks/b/bobhelper.rb:15]:
```suggestion
  zap trash: [
    "~/Library/Application Support/com.hezongyidev.BobHelper",
    "~/Library/Caches/com.hezongyidev.BobHelper",
    "~/Library/HTTPStorages/com.hezongyidev.BobHelper",
    "~/Library/Preferences/com.hezongyidev.BobHelper.plist",
    "~/Library/Saved Application State/com.hezongyidev.BobHelper.savedState",
  ]
```

**Review by khipp — COMMENTED** [review]
Thank you for your contribution, @mogeko! I left a few suggestions below.

**Comment by mogeko** [comment]
@khipp Thank you for your excellent suggestion! please review~

**Review by khipp — APPROVED** [review]
Thank you for your first contribution to `homebrew-cask`, @mogeko! 🎉

**Pull request merged**